### PR TITLE
Add recruitment banner to guides and answers

### PR DESCRIPTION
**Pull request by hannako**
Add recruitment banner to pages of interest as requested by AI team.

[Trello](https://trello.com/c/vnr9mW0w/3025-put-live-govuk-user-research-banner-requests-ai-govuk)

Review app links

**Guides**
- https://government-frontend-pr-3412.herokuapp.com/self-assessment-tax-returns 
- https://government-frontend-pr-3412.herokuapp.com/self-employed-records
- https://government-frontend-pr-3412.herokuapp.com/expenses-if-youre-self-employed
- https://government-frontend-pr-3412.herokuapp.com/first-company-accounts-and-return
- https://government-frontend-pr-3412.herokuapp.com/capital-allowances
- https://government-frontend-pr-3412.herokuapp.com/simpler-income-tax-cash-basis
- https://government-frontend-pr-3412.herokuapp.com/capital-gains-tax
- https://government-frontend-pr-3412.herokuapp.com/directors-loans
- https://government-frontend-pr-3412.herokuapp.com/running-a-limited-company
- https://government-frontend-pr-3412.herokuapp.com/introduction-to-business-rates
- https://government-frontend-pr-3412.herokuapp.com/apply-for-business-rate-relief
- https://government-frontend-pr-3412.herokuapp.com/tax-codes

**Answers**

- https://government-frontend-pr-3412.herokuapp.com/working-for-yourself
- https://government-frontend-pr-3412.herokuapp.com/what-is-the-construction-industry-scheme
- https://government-frontend-pr-3412.herokuapp.com/expenses-and-benefits-a-to-z
- https://government-frontend-pr-3412.herokuapp.com/self-assessment-tax-return-forms
- https://government-frontend-pr-3412.herokuapp.com/calculate-tax-on-company-cars
- https://government-frontend-pr-3412.herokuapp.com/calculate-your-business-rates
- https://government-frontend-pr-3412.herokuapp.com/stop-being-self-employed

### CODE REVIEW

File: lib/data/recruitment_banners.yml [hunk 1]
```diff
@@ -16,3 +16,27 @@ banners:
...
+    - /capital-gains-tax
+    - /directors-loans
+    - /self-assessment-tax-return-forms
+    - /running-a-limited-company
+    - /calculate-tax-on-company-cars
+    - /introduction-to-business-rates
+    - /calculate-your-business-rates
+    - /apply-for-business-rate-relief
+    - /stop-being-self-employed
+    - /tax-codes
```
Comment (catalinailie) [inline lib/data/recruitment_banners.yml:42]:
```suggestion
    - /tax-codes
    
```

Comment (hannako) [reply lib/data/recruitment_banners.yml:42]: I can't spot the issue here??

Comment (catalinailie) [reply lib/data/recruitment_banners.yml:42]: It's not a blocker. The file is missing the new line at the end.

**Review by catalinailie — APPROVED** [review]

File: lib/data/recruitment_banners.yml (hunk 1 above)
Comment (hannako) [reply lib/data/recruitment_banners.yml:42]: Ah ok! Thank you!

**Review by catalinailie — APPROVED** [review]

**Pull request merged**